pkg/config/user_config.go

```diff
@@ -570,7 +608,7 @@ func GetDefaultConfig() *UserConfig {
 				BulkMenu: "b",
 			},
 		},
-		OS:                           GetPlatformDefaultConfig(),
+		OS:                           OSConfig{},
```

Comment:
For posterity I'll just state that (if I understand correctly) we're not using the default here because it prevents us from knowing whether a user has explicitly set one of the configs or not.

Reply:
Exactly, that's how I tried to explain it in the commit message. 😉